### PR TITLE
Fix footer overflow on larger screens and adjust text and icon sizes …

### PR DESCRIPTION
**Pull request by Arpin007**
#216  I've fixed the issue related to the UI, and it looks much better now!

image

@AlfiyaSiddique Please check & marged the pull request please also add lavel before margin the same...

### CODE REVIEW

**Comment by AlfiyaSiddique** [comment]
@Arpin007 make the contributor's text a bit smaller as it was before. Plesae take a look at the prod deployed link.

It is still overflowing here:
image

Also, can you fix the mobile size alignment:
image

**Comment by Arpin007** [comment]
@AlfiyaSiddique It is taking time to fix the responsive mobile size alignment, will fix by today...

Thankyou...

**Comment by Arpin007** [comment]
@AlfiyaSiddique Have made the changes now the UI is responsive on mobile devices also, Please review & confirm if anything needs to be changed if it is ok now then please marge and add levels please try for level 3 😊😊

**Comment by AlfiyaSiddique** [comment]
@Arpin007 it is still not fixed: Size -> 1768 X 682 refer deploy preview for this
image

Also please revert the changes you made for the contribution text and icons.
Make it as it is in this link: https://delightful-daifuku-a9f6ea.netlify.app/

**Comment by Arpin007** [comment]
@AlfiyaSiddique Have reverted the changes made for the contribution text and icons

File: backend/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@octokit/rest": "^21.0.2",
     "axios": "^1.6.3",
+    "backend": "file:",
     "bcrypt": "^5.1.1",
```
Comment (AlfiyaSiddique) [inline backend/package.json:18]: @Arpin007 why did you added the backend dependency. It is not used anywhere.
Can you remove it if not required.

File: frontend/package.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@react-oauth/google": "^0.12.1",
     "axios": "^1.6.3",
+    "frontend": "file:",
     "install": "^0.13.0",
```
Comment (AlfiyaSiddique) [inline frontend/package.json:21]: Same reason as above, please remove it if not required @Arpin007

**Comment by AlfiyaSiddique** [comment]
@Arpin007 the output looks good, just some comments on your added code. Please fix it and I will merge this.
Also would like to mention that this change could not get level3 label as it is for adding new features only:)
But don't worry I will add labels fairly.

**Comment by Arpin007** [comment]
@AlfiyaSiddique I wasn't allowed to push the code because I had only removed the unused dependency, so I also corrected the Select Language button border to match the Feedback button 

please review now aslo the dependency was add during npm install sorry for that..

**Comment by Arpin007** [comment]
@AlfiyaSiddique Have removed the comments please check also have no issue with the levels...

**Comment by AlfiyaSiddique** [comment]
@Arpin007 the dependencies are still there you can check in the Files changed section above.
you can run  `npm uninstall backend`  or `npm uninstall frontend`in their respective directories to remove them and push.

**Comment by Arpin007** [comment]
@AlfiyaSiddique Done please check now....

**Review by AlfiyaSiddique — APPROVED** [review]

**Comment by AlfiyaSiddique** [comment]
Looks good @Arpin007 
Thanks for the contribution

**Comment by Arpin007** [comment]
Welcome😊😊

**Pull request merged**